goss-testing/scripts/k8s_psp_enabled_check.sh

```diff
@@ -0,0 +1,75 @@
+#!/usr/bin/env bash
```

Comment:
nit: would recommend just using `#!/bin/bash`, it is very nuance to have more than 1 BASH shell installed.